addon/components/menu/items.hbs

```diff
@@ -2,7 +2,9 @@
   <div
     id={{@guid}}
     aria-labelledby={{@buttonGuid}}
+    aria-activedescendant={{this.activeDescendantId}}
```

Comment:
perhaps we could avoid the indirection and do:

`aria-activedescendant=@activeItem.element.id`

Reply:
😅 I had that originally and replaced it before actually testing it out, thinking it wouldn't work if `@activeItem` isn't defined. Property access in templates automatically go through `Ember.get` though (IRRC) so that should be safe after all!